packages/nc-gui/utils/validation.ts

```diff
@@ -98,13 +98,15 @@ export const fieldRequiredValidator = () => {
   }
 }
 
-export const fieldLengthValidator = (sqlClientType: string) => {
+export const fieldLengthValidator = (_sqlClientType: string) => {
```

Comment:
`_sqlClientType: string` can be removed. we no longer need that. double check the caller.